openshift/buildConfig-template.yaml

```diff
@@ -40,12 +40,86 @@ parameters:
     value: "latest"
 
 objects:
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: solver-fedora-32-py38
+      annotations:
+        thoth-station.ninja/template-version: 1.7.0
+      labels:
+        app: thoth
+        component: solver
+        app.kubernetes.io/name: "thoth"
+        app.kubernetes.io/component: "solver"
+        app.thoth-station.ninja/solver: "fedora-32-py38"
+    spec:
+      output:
+        to:
+          kind: ImageStreamTag
+          name: "solver-fedora-32-py38:${IMAGE_STREAM_TAG}"
+      source:
+        dockerfile: |
+          FROM fedora:32
```

Comment:
Marking as WIP as this image is not available yet - we should base our solvers on s2i container images as they have some additional dependencies as described in [2].

[1] https://github.com/sclorg/s2i-python-container/pull/356/files#diff-f2f92a6c82897585e4b843237db8b435R44
[2] https://github.com/thoth-station/solver/issues/417